crates/ruff_cli/tests/integration_test.rs

```diff
@@ -112,7 +112,8 @@ fn test_stdin_json() -> Result<()> {
       "row": 1,
       "column": 10
     }},
-    "filename": "{file_path}"
+    "filename": "{file_path}",
+    "noqa_line": 1
```

Comment:
Now that this is part of the public API, I'm wondering if it should be `noqa_row`, for consistency with `row` in `location`. What do you think?

Reply:
Makes sense. I'll change it in `Message` and the JSON; if you'd like me to change all of the existing places as well, can do that as well.

Reply:
I think it'd be best to change it everywhere, but I'd be fine merging this with just the public API changes.

Reply:
OK, made the API change, will send a PR doing the internal renaming later.